src/components/DryRunnerBrawlModifiers/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Dialog from '../Dialog'
+
+export default React.memo(function DryRunnerBrawlModifiers(props) {
+  const dialogRef = React.useRef()
+  const modifierRef = React.useRef()
```

Comment:
Right, I see how you use it, but I think you should be able to store that in a local state.
```js
const [modifier, setModifier] = React.useState(props.modifier)
```
```js
onClick: () => {
  props.setModifier(modifier)
  dialogRef.current.hide()
}
```